docs/profiles.md

```
@@ -0,0 +1,166 @@
+root_path = "/opt/soar"
+```
+
+```bash
+sudo soar --system --profile global install git
+```
+
+## Directory Structure
+
+Profile with `root_path = "~/.local/share/soar"`:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)